state/sequence.go

```diff
@@ -57,13 +56,9 @@ func (st *State) sequence(name string) (int, error) {
 // `sequence` is more efficient than `sequenceWithMin` and should be
 // preferred if there is no minimum value requirement.
 func (st *State) sequenceWithMin(name string, minVal int) (int, error) {
-	sequences, closer := st.getCollection(sequenceC)
+	sequences, closer := st.getRawCollection(sequenceC)
```

Comment:
The existing tests already check multi-model behaviour. This is an internal only change.